awscfn/table_awscfn_resource.go

```diff
@@ -0,0 +1,228 @@
+package awscfn
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/awslabs/goformation/v6"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"gopkg.in/yaml.v3"
+)
+
+func tableAWSCFNResource(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "awscfn_resource",
+		Description: "Cloudformation resource information.",
+		List: &plugin.ListConfig{
+			Hydrate:    listAWSCloudFormationResources,
+			KeyColumns: plugin.OptionalColumns([]string{"path"}),
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "An identifier for the resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "type",
+				Description: "The resource type identifies the type of resource that you are declaring.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "literal_value",
```

Comment:
```suggestion
				Name:        "properties_src",
```

I think this column name is more consistent with what we use in other tables, e.g., `tags` and `tags_src`, and makes it easier to tell the `properties` and `properties_src` columns have similar information 

Please update any references to this column as well